server.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+const net = require("net");
+const files = require("./files.js");
+
+const PORT = 8080;
+let date = new Date();
```

Comment:
You should move this inside the createServer function. The way it is, if you're server runs for a long time, it'll always send back the date which the server was started.